scss/5-objects/_page-content.scss

```diff
@@ -0,0 +1,15 @@
+// ============================================================================
+// Page content
+// ============================================================================
+
+/** @define page-title */
+.cads-page-title {
+  @extend %cads-h1;
+
+  margin-top: $cads-spacing-5;
+}
+
+/** @define page-content */
+.cads-page-content {
```

Comment:
I could go one step further and make this a margin utility class? But this seemed like a reasonable middle ground as this ~80px margin is specific for the bottom of page.